src/ToDo.iOS/Info.plist

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-  <dict>
+<dict>
 	<key>CFBundleDisplayName</key>
 	<string>ToDo</string>
 	<key>CFBundleIdentifier</key>
-	<string>com.companyname.ToDo</string>
+	<string>com.horus.unoplatform.todoapp</string>
```

Comment:
@jeromelaban can you let the team know what the bundle name should be before this gets merged